NASA-AMMOS/aerie-ui
# Feature/add retrigger model extraction

Resolves #1326 

Depends on:

Aerie Gateway endpoint: https://github.com/NASA-AMMOS/aerie-gateway/pull/93
Aerie Backend: https://github.com/NASA-AMMOS/aerie/pull/1467

To test:
1. Go to model creation page
2. Stop the aerie_merlin container
3. Create a model
4. Verify that an error is displayed for the newly created model
5. Start up the aerie_merlin container
6. After aerie_merlin has started up, click on the refresh icon next to the "Jar file status"
![Screenshot 2024-06-25 at 8 33 51 AM](https://github.com/NASA-AMMOS/aerie-ui/assets/5290214/326e8c91-daa1-4f61-a28d-6faea6788c1a)
7. Verify that the error on the model goes away
